src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/BlackDuckPolicyCollector.java

```diff
@@ -22,50 +22,95 @@
  */
 package com.synopsys.integration.alert.provider.blackduck.collector;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.HashMap;
 import java.util.List;
-import java.util.Set;
-import java.util.SortedSet;
-import java.util.TreeSet;
-import java.util.stream.Collectors;
+import java.util.Map;
+import java.util.Optional;
 
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.synopsys.integration.alert.common.enumeration.ComponentItemPriority;
 import com.synopsys.integration.alert.common.enumeration.ItemOperation;
-import com.synopsys.integration.alert.common.message.model.CategoryItem;
-import com.synopsys.integration.alert.common.message.model.CategoryKey;
+import com.synopsys.integration.alert.common.message.model.ComponentItem;
 import com.synopsys.integration.alert.common.message.model.LinkableItem;
 import com.synopsys.integration.alert.common.provider.ProviderContentType;
 import com.synopsys.integration.alert.common.workflow.filter.field.JsonExtractor;
 import com.synopsys.integration.alert.common.workflow.processor.MessageContentProcessor;
 import com.synopsys.integration.alert.provider.blackduck.BlackDuckProperties;
+import com.synopsys.integration.alert.provider.blackduck.descriptor.BlackDuckContent;
+import com.synopsys.integration.blackduck.api.manual.component.PolicyInfo;
 
 public abstract class BlackDuckPolicyCollector extends BlackDuckCollector {
-    public static final String CATEGORY_TYPE = "policy";
+    public static final String CATEGORY_TYPE = "Policy";
+    private final Map<String, ComponentItemPriority> priorityMap = new HashMap<>();
+    private Logger logger = LoggerFactory.getLogger(getClass());
 
     public BlackDuckPolicyCollector(final JsonExtractor jsonExtractor, final List<MessageContentProcessor> messageContentProcessorList, final Collection<ProviderContentType> contentTypes, final BlackDuckProperties blackDuckProperties) {
         super(jsonExtractor, messageContentProcessorList, contentTypes, blackDuckProperties);
+
+        priorityMap.put("blocker", ComponentItemPriority.HIGH);
```

Comment:
What ends up being displayed in the end? Blocker or High? The customers are going to want to see the provider specified value, not our interpretation of it. And the ComponentItemPriority needs to be expanded to support more granularity. Having Major policy violations show up above Blockers will not go over well

Reply:
The priority is only used for sorting.

Reply:
Severity is added as its own linkable item.